.changeset/rude-books-rush.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/backend-tasks': patch
+'@backstage/plugin-app-backend': patch
+'@backstage/plugin-bazaar-backend': patch
+'@backstage/plugin-code-coverage-backend': patch
+'@backstage/plugin-scaffolder-backend': patch
+'@backstage/plugin-search-backend-module-pg': patch
+'@backstage/plugin-tech-insights-backend': patch
```

Comment:
For `scaffolder-backend` and `search-backend-module-pg` these are technically breaking changes. This means that those two packages should have separate changesets, one each, explaining the change and telling people what they need to do in their backend code to migrate to using the new parameters.

For `search-backend-module-pg` you would bump to `minor` since it's 0.x still.

For `scaffolder-backend` it's worse; that's a 1.x stable package so it would formally be a `major` change and we really don't want that. I would suggest that your `DatabaseTaskStoreOptions` to accept `Knex | PluginDatabaseManager` so that it's backwards compatible, and change your code to handle whichever comes in. Then the changeset could be just `minor` instead.

Reply:
Perfect. In the case of the scaffolder, as you suggested, I tried to make a code that supports both the old and the new version. Do you have any concerns about it?